week1/armstrong.rb

```diff
@@ -9,5 +9,25 @@
 # and `false` otherwise.
 
 def armstrong_number(n)
-    raise NotImplementedError # Delete this line and start coding here
+    
+    if n < 0
+        return false
+        
+    else
+
+        l = n.digits
+        sum = 0
+        for i in l do
+            sum += i**3
```

Comment:
the problem statement asks for sum of digits raised to number of digits in the number. So this `3` should be replaced by `l.length` (`sum += i**l.length`). make this one change and push again